jans-cedarling/cedarling/src/authz/entity_builder/build_expr.rs

```diff
@@ -29,62 +30,106 @@ impl Attribute {
     /// Builds a [`RestrictedExpression`] while checking the schema
     pub fn build_expr(
         &self,
-        attr_src: &HashMap<String, Value>,
         src_key: &str,
+        attr_claim_value_opt: Option<&Value>,
```

Comment:
I don't think we need to add `_opt` here since the `Option` type already does that for us.
```suggestion
        attr_claim_value: Option<&Value>,
```